test/tests/controls/tag_based_naming_access/object_param.i

```diff
@@ -0,0 +1,90 @@
+[Mesh]
+  type = GeneratedMesh
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  elem_type = QUAD4
+  uniform_refine = 4
+[]
+
+[Variables]
+  [./diffused]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = Diffusion
+    variable = diffused
+  [../]
+[]
+
+[DiracKernels]
+  [./test_object]
+    type = MaterialPointSource
+    point = '0.5 0.5 0'
+    variable = diffused
+    control_tags = 'tag'
```

Comment:
Can I suggest that maybe you name your tag here something other than `tag`?  We're going to point users at it and it's a bit confusing the way it is.

Name it `fred` or `george` or whatever you want... but `tag` is a bit confusing because when you use it you do `tag::point` like that... and it's not 100% clear that `tag` is just a name you've chosen at random instead of being something you HAVE to use.